cmd/opentelemetry/app/defaultconfig/default_config.go

```diff
@@ -105,19 +107,11 @@ func createReceivers(component ComponentType, zipkinHostPort string, factories c
 	jaeger := factories.Receivers["jaeger"].CreateDefaultConfig().(*jaegerreceiver.Config)
 	// The CreateDefaultConfig is enabling protocols from flags
 	// we do not want to override it here
-	if _, ok := jaeger.Protocols["grpc"]; !ok {
-		jaeger.Protocols["grpc"] = &receiver.SecureReceiverSettings{
-			ReceiverSettings: configmodels.ReceiverSettings{
-				Endpoint: gRPCEndpoint,
-			},
-		}
+	if jaeger.GRPC == nil {
+		jaeger.GRPC = &configgrpc.GRPCServerSettings{}
```

Comment:
Are these (this line and the same for the other Jaeger protocols) missing the Jaeger specific endpoints?

Reply:
Correct they should be provided here